default/scripting/species/SP_KHAKTURIAN.focs.txt

```diff
@@ -0,0 +1,72 @@
+Species
+    name = "SP_KHAKTURIAN"
+    description = "SP_KHAKTURIAN_DESC"
+    gameplay_description = "SP_KHAKTURIAN_GAMEPLAY_DESC"
+    Native
+    CanProduceShips
+
+    // Desert: Khakturian: Planetbound, Great Attack Troops, Great Defense Troops, Phototrophic
+
+    tags = [ "PHOTOTROPHIC" "BAD_POPULATION" "BAD_INDUSTRY" "NO_RESEARCH" "BAD_INFLUENCE" "GOOD_STOCKPILE" "GREAT_ATTACKTROOPS" "PEDIA_PHOTOTROPHIC_SPECIES_CLASS" ]
+
+    foci = [
+        [[HAS_INDUSTRY_FOCUS]]
+        /*[[HAS_RESEARCH_FOCUS]]*/    // dumb plants after all
+        /*[[HAS_INFLUENCE_FOCUS]]*/   // see above
+        [[HAS_GROWTH_FOCUS]]
+        [[HAS_ADVANCED_FOCI]]
+    ]
+    defaultfocus = "FOCUS_GROWTH"
```

Comment:
growth focus does not make sense gameplay-wise for a species which cant colonize, it only boosts population on non-homeworlds. so if this is not intentionally for making invasion easier, rather default to protection.

Reply:
can do. just seemed fitting for plants, they like to grow.